src/hooks/useLocale.jsx

```diff
@@ -0,0 +1,23 @@
+// import { useState, useEffect } from "react";
+import Spanish from "../lang/es-US.json";
+import English from "../lang/en-US.json";
+
+// detect the language preference of the user's browser
+const useLocale = () => {
+  const locale = window.navigator.userLanguage || window.navigator.language;
+  // const [messages, setMessages] = useState(English);
+
+  // useEffect(() => {
```

Comment:
❓ Is it possible to bring `useEffect` back?

Reply:
Addressed:  b00c88249d441a7b6f25d3d112c8f5d08b4bd7bd